Update german translation of documentation

M4SSD35TRUCT10N opened the pull request:
Update german translation of documentation WIP

- [x] Welcome: index.md
- [x] First Steps:
  - [x] Setup: install.md
  - [ ] Portable App: portable.md
  - [ ] Installing LaTeX: installing-latex.md
  - [ ] Getting Started: get-started.md
  - [x] Get involved: get-involved.md
  - [x] FAQ: faq.md
  - [ ] Troubleshooting: troubleshooting.md
- [x] Concepts:
  - [x] File management: concepts/workspaces.md
  - [x] Context: concepts/context.md
  - [x] WYSIWYM: concepts/wysiwym.md
- [x] Core:
  - [x] Editor: core/editor.md
  - [x] Document Tabs: core/tabs.md
  - [x] File Manager: core/file-manager.md
  - [x] Assets Manager: core/assets-manager.md
  - [x] Defaults Files: core/defaults-files.md
  - [x] Custom CSS: core/custom-css.md
  - [x] Snippets: core/snippets.md
  - [x] AutoCorrect and MagicQuotes: core/autocorrect.md
  - [x] Searching: core/search.md
  - [x] Importing: core/import.md
  - [x] Exporting: core/export.md
  - [x] Table Editor: core/tables.md
  - [x] Localisation: core/localisation.md
  - [x] Sidebar: core/sidebar.md
  - [x] YAML Frontmatter: core/yaml-frontmatter.md
  - [x] Print and Preview: core/print-preview.md
  - [x] Math: core/math.md
- [x] Academic Tools:
  - [x] Citations: academic/citations.md
  - [x] Projects: academic/projects.md
  - [x] Presentations: academic/presentations.md
  - [x] Pomodoro Timer: academic/pomodoro.md
  - [x] Readability: academic/readability.md
  - [x] Zettelkasten Methods: academic/zkn-method.md
  - [x] Graph View: academic/graph.md
  - [x] Custom Templates: academic/custom-templates.md
- [ ] References:
  - [ ] Keyboard Shortcuts: reference/keyboard-shortcuts.md
  - [ ] Markdown Basics: reference/markdown-basics.md
  - [ ] Settings: reference/settings.md
  - [ ] Spell Checking: reference/spell-checking.md
- [ ] informal language and overall polishing

M4SSD35TRUCT10N committed:
Add graph view documentation

Signed-off-by: Enrico Lefass <M4SSD35TRUCT10N@users.noreply.github.com>

M4SSD35TRUCT10N committed:
Translate german presentations.md

The german translation of presentation.md was updated, since it was effectively a copy of the english documentation.

M4SSD35TRUCT10N committed:
Update german pomodoro.md

Update the german pomodoro.md as it was effectively a copy of the english one.

M4SSD35TRUCT10N committed:
Update de.yml

Change description of citations to matching first headline in the document.

M4SSD35TRUCT10N committed:
Update german translation of readability.md

Updating the german readabilityy.md since it was effectively a copy of the english documentation.

M4SSD35TRUCT10N committed:
Fix link in readability.md

Fix the link to the https://zettlr.com/readability.

M4SSD35TRUCT10N committed:
Update german translation of customm-templates.md

Update german translation of customm-templates.md as it was effectively a copy of the english documentation.

M4SSD35TRUCT10N committed:
Update de.yml

Update de.yml matching the first header of the german translation in custom-templates.md.

M4SSD35TRUCT10N committed:
Update de.yml

Replacing core with Kern.

M4SSD35TRUCT10N committed:
Add german translation of concepts

Add german translation of concepts with context.md, workspaces.md and wysiwym.md.

nathanlesage commented:
In general looks quite well. I stumbled upon the formal addressing ("Sie"), since the rest of the documentation refers to the informal "du". Therefore, it would be great if you could rephrase the sentences to use informal language.

Then, a few sentences looked a little bit awkward and too close to the English, so feel free to take some freedom of adapting the sentences, which will clearly make the documentation sound smoother.

Lastly, I think the custom templates section was very outdated, since we don't have all of that old templating system anymore. It would be great if you could instead translate from the current, up-to-date English documentation [here](https://docs.zettlr.com/en/academic/custom-templates/).

Thank you already for your efforts!

M4SSD35TRUCT10N commented:
> In general looks quite well. I stumbled upon the formal addressing ("Sie"), since the rest of the documentation refers to the informal "du". Therefore, it would be great if you could rephrase the sentences to use informal language.
> 
> Then, a few sentences looked a little bit awkward and too close to the English, so feel free to take some freedom of adapting the sentences, which will clearly make the documentation sound smoother.
> 
> Lastly, I think the custom templates section was very outdated, since we don't have all of that old templating system anymore. It would be great if you could instead translate from the current, up-to-date English documentation [here](https://docs.zettlr.com/en/academic/custom-templates/).
> 
> Thank you already for your efforts!

Will do the rest and then change to informal addressing with one commit. The Templates section was taken from the already existing de/custom-templates.md. Will update that maybe this evening. Awkwardness of some translated sentences: I know. I use to tend to get things done in the first place and polish it later. :-)

Seems that I can finish the first version of german translation this weekend. Sunday evening I guess.

Thank you for your advices! Greeting from Wildau, Germany.

nathanlesage commented:
Sounds great!

M4SSD35TRUCT10N committed:
Update german translation of custom-templates.md

Update german translation of custom-templates.md to catch up with english version.

M4SSD35TRUCT10N committed:
Fix typo in graph.md

Fix typo in graph.md and add missing png.

M4SSD35TRUCT10N committed:
Add missing png

Add missing png for custom-templates.md.

M4SSD35TRUCT10N committed:
Update german guides

Update the german translation of the guidessection as it was essentially in english. guide-ide.md and guides-notes.md were translated so the sectionis now complete.

M4SSD35TRUCT10N commented:
I've updated the comment to reflect the current status. Since I've updated from the english documentation (copied files over) the for instance the core section is now a reset to english. Hopefully I find time this week to attack the left files and translate them. Sorry for the delay.

M4SSD35TRUCT10N committed:
Add german translation of core

Add german translation of documentation of the core.

nathanlesage commented:
Are there any news on this PR? There are currently merge conflicts that would need to be fixed prior to merging.

M4SSD35TRUCT10N commented:
> Are there any news on this PR? There are currently merge conflicts that would need to be fixed prior to merging.

Due to a disk drive malfunction and a heavy workload the last 4 years I didn't had any chance to resolve this. Will try to start up the old laptop in hope there is an old staged version.

nathanlesage commented:
In case you can't find anything, you can also simply clone the work you did already from here, and continue there. Please feel free to close this and open a new PR if required. Looking forward to your help!

nathanlesage commented:
Closing due to inactivity.
